@copilot fix checkstyle errors:


```
Error: eckstyle] [ERROR] /home/runner/work/Hoptimator/Hoptimator/hoptimator-k8s/src/test/java/com/linkedin/hoptimator/k8s/K8sDatabaseDeployerTest.java:6:8: Unused import - com.linkedin.hoptimator.k8s.models.V1alpha1DatabaseSpec. [UnusedImports]
```

--- a/hoptimator-k8s/src/test/java/com/linkedin/hoptimator/k8s/K8sDatabaseDeployerTest.java
+++ b/hoptimator-k8s/src/test/java/com/linkedin/hoptimator/k8s/K8sDatabaseDeployerTest.java
@@ -3,7 +3,6 @@ package com.linkedin.hoptimator.k8s;
 import com.linkedin.hoptimator.DatabaseDeployable;
 import com.linkedin.hoptimator.k8s.models.V1alpha1Database;
 import com.linkedin.hoptimator.k8s.models.V1alpha1DatabaseList;
-import com.linkedin.hoptimator.k8s.models.V1alpha1DatabaseSpec;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 

